### PR TITLE
chore(project): Bump to 7.22

### PR DESCRIPTION
**Pull request by psavidis**
- Bump to latest JUnit & AssertJ
- Remove obsolete telemetry config

Related-to: https://github.com/camunda/camunda-bpm-platform/issues/4267

### CODE REVIEW

File: src/test/resources/camunda.cfg.xml [hunk 1]
```diff
@@ -18,9 +18,6 @@
     <property name="jobExecutorActivate" value="false" />
 
     <property name="history" value="full" />
-    
-    <!-- telemetry configuration -->
-    <property name="telemetryReporterActivate" value="false" />
```
Comment (danielkelemen) [inline src/test/resources/camunda.cfg.xml:23]: 👍 Nice

File: pom.xml [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
 
     <!-- set the version for Camunda Platform here -->
-    <version.camunda>7.21.0</version.camunda>
-    <version.junit>5.10.2</version.junit>
+    <version.camunda>7.22.0</version.camunda>
+    <version.junit>5.11.0</version.junit>
     <version.h2>2.1.214</version.h2>
-    <version.assertj>3.25.3</version.assertj>
+    <version.assertj>3.26.3</version.assertj>
     <version.slf4j>1.7.36</version.slf4j>
```
Comment (danielkelemen) [inline pom.xml:16]: 🔧

```suggestion
    <version.slf4j>2.0.16</version.slf4j>
```

File: pom.xml [hunk 3]
```diff
@@ -9,10 +9,10 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- set the version for Camunda Platform here -->
-    <version.camunda>7.21.0</version.camunda>
-    <version.junit>5.10.2</version.junit>
+    <version.camunda>7.22.0</version.camunda>
+    <version.junit>5.11.0</version.junit>
     <version.h2>2.1.214</version.h2>
```
Comment (danielkelemen) [inline pom.xml:14]: 🔧

```suggestion
    <version.h2>2.3.232</version.h2>
```

**Review by danielkelemen — APPROVED** [review]
👍

**Comment by psavidis** [comment]
### Next step
Merge after the `7.22.0` is released.

**Comment by psavidis** [comment]
Tested the PR against `7.22.0` and passed successfully.

Proceeding to merge.

**Pull request merged**